components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -333,6 +333,10 @@ class GeckoEngine(
                     runtime.settings.fontSizeFactor = it
                 }
             }
+
+        override var forceUserScalableContent: Boolean
+            get() = runtime.settings.forceUserScalableEnabled
+            set(value) { runtime.settings.forceUserScalableEnabled = value }
     }.apply {
         defaultSettings?.let {
```

Comment:
We will need to add this to the default setting block below, so that a provided default value is applied, see: https://github.com/mozilla-mobile/fenix/blob/8c30200ced8e60756829518170fc8e53e3eb650b/app/src/main/java/org/mozilla/fenix/components/Core.kt#L51

There's a separate test to make sure the default settings are applied which needs an update to verify that `forceUserScalableContent` is applied: https://github.com/mozilla-mobile/android-components/blob/master/components/browser/engine-gecko-nightly/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineTest.kt#L308